packages/langium/src/lsp/hover-provider.ts

```diff
@@ -79,15 +96,43 @@ export class MultilineCommentHoverProvider extends AstNodeHoverProvider {
         return undefined;
     }
 
-    protected getCommentContent(commentText: string): string {
-        const split = commentText.split('\n').map(e => {
-            e = e.trim();
-            if (e.startsWith('*')) {
-                e = e.substring(1).trim();
+    protected getAdditionalHoverContent(_node: AstNode): string | undefined {
+        return undefined;
+    }
+
+    protected hoverLinkRenderer(node: AstNode, name: string, display: string): string | undefined {
+        const description = this.findNameInPrecomputedScopes(node, name) ?? this.findNameInGlobalScope(node, name);
```

Comment:
I fear that this would force some users who have heavily customized the scope provider to apply similar customizations here as well. Could you try reusing the scope provider service here? The `property` can be set to an exported constant like `$jsdoc`, so language implementers can add special case handling in their scope provider if they want.

Reply:
Not sure how this would work. The scoping algorithm always expects that we have some kind of reference type. With the current scoping implementation, the reflection will just always throw.

I'm sure there's a clever solution for this somewhere, but I believe it's out of scope for this PR. The hover implementation is just a showcase on how this could be used. We can improve it later on.